tests/operator_test.go

```diff
@@ -2284,7 +2284,12 @@ spec:
 			checks.SkipIfVersionBelow("Skipping dynamic cdi test in versions below 1.13 because crd garbage collection is broken", "1.13")
 
 			// This tests starting infrastructure with and without the DataVolumes feature gate
-			vm = tests.NewRandomVMWithDataVolume(cd.DataVolumeImportUrlForContainerDisk(cd.ContainerDiskAlpine), util2.NamespaceTestDefault)
+			var foundCS bool
+			vm, foundCS = tests.NewRandomVMWithDataVolume(cd.DataVolumeImportUrlForContainerDisk(cd.ContainerDiskAlpine), util2.NamespaceTestDefault)
```

Comment:
Can you use `:=` here? And name it `foundSC` (typo?)

Reply:
Can't use `"=` here. It will cause shadoeing of the`vm` variable.